src/models/externals/IExternalRoutePathNode.ts

```diff
@@ -0,0 +1,15 @@
+import IExternalRoutePath from './IExternalRoutePath';
+
+export default interface IExternalRoutePathNode {
+    geojson: string;
+    reitinlinkkisByReitunnusAndSuuvoimastAndSuusuunta: {
+        nodes: IExternalRoutePath[],
+    };
```

Comment:
this doesn't seem right. Yes, the query returns object with field "reitinlinkkisByReitunnusAndSuuvoimastAndSuusuunta" but it could be changed to just { routePaths: IExternalRoutePath[] } at our frontend.

Reply:
Btw, this is called IExternalRoutePath_Node_ but it has IExternalRoutePath[] array inside. How node can have relation to routePaths?

Our model relation tree is: route -> routePaths -> routePathLinks -> routePathLinkNodes